### DISCUSSION TITLE
CSRF Vulnerability

### DISCUSSION THREAD

**Discussion by keniquestores** [requarks/wiki#7234] [Help / Questions] [answered]
Regarding the CSRF vulnerability, is there something implemented for this security flaw?

**Comment by NGPixel** [comment]
What vulnerability?

**Reply by keniquestores** [reply]
![image](https://github.com/requarks/wiki/assets/129752821/72cd79ae-3a51-4931-9543-cce4995450c6)

All of our applications that are in our domain are pentested to validate possible vulnerabilities, whether using Zep Scanning or the Score Card. They accuse: Absence of Anti-CSRF tokens.
No Anti-CSRF tokens were found in the HTML submission form.

**Reply by NGPixel** [reply, marked as answer]
A CSRF token is not really relevant in this case because of how Wiki.js works with its API. It could be revisited in the future for extra security but I don't see it as a vulnerability at the moment.
